src/sentry/apidocs/hooks.py

```diff
@@ -220,91 +223,106 @@ def dereference_schema(schema: dict, schema_components: dict) -> dict:
-                    # media type can either "multipart/form-data" or "application/json"
-                    if "multipart/form-data" in content:
-                        schema = content["multipart/form-data"]["schema"]
-                    else:
-                        schema = content["application/json"]["schema"]
-
-                    # Dereference schema if needed
-                    schema = dereference_schema(schema, schema_components)
-
-                    for body_param, param_data in schema["properties"].items():
-                        # Ensure body parameters have a description. Our API docs don't
-                        # display body params without a description, so it's easy to miss them.
-                        # We should be explicitly excluding them as better
-                        # practice however.
-
-                        # There is an edge case where a body param might be
-                        # reference that we should ignore for now
-                        if "description" not in param_data and "$ref" not in param_data:
-                            raise SentryApiBuildError(
-                                f"""Body parameter '{body_param}' is missing a description for endpoint {endpoint_name}. You can either:
-                            1. Add a 'help_text' kwarg to the serializer field
-                            2. Remove the field if you're using an inline_serializer
-                            3. For a DRF serializer, you must explicitly exclude this field by decorating the request serializer with
-                            @extend_schema_serializer(exclude_fields=[{body_param}])."""
-                            )
-
-                    # Required params are stored in a list and not in the param itself
-                    required = set(schema.get("required", []))
-                    if required:
-                        # Explicitly sort body params by converting the dict to an ordered dict
-                        schema["properties"] = OrderedDict(
-                            sorted(
-                                schema["properties"].items(),
-                                key=lambda param: 0 if param[0] in required else 1,
-                            )
-                        )
-                except KeyError as e:
-                    raise SentryApiBuildError(
-                        f"Unable to parse body parameters due to KeyError {e} for endpoint {endpoint_name}. Please post in #discuss-apis to fix."
-                    )
+            try:
+                requestBody = method_info.get("requestBody")
+                if requestBody is not None:
+                    _validate_request_body(requestBody, schema_components, endpoint_name)
+            except KeyError as e:
+                raise SentryApiBuildError(
+                    f"Unable to parse body parameters due to KeyError {e} for endpoint {endpoint_name}. Please post in #discuss-apis to fix."
```

Comment:
While you're at it fix this to `discuss-api` singular? 😅

Reply:
lol ok